Always use HTL style of commenting

 == Noncompliant Code Example

 ``
    <!-- Potentially bad, plain HTML comments will be visible on the page -->
``

 == Compliant Solution

 ``
    <!--/* Good, this comment will not be rendered */-->
``